tests/unit/grains/test_core.py

```diff
@@ -605,6 +605,26 @@ def test_ubuntu_artful_os_grains(self):
         }
         self._run_os_grains_tests("ubuntu-17.10", _os_release_map, expectation)
 
+    @skipIf(not salt.utils.platform.is_linux(), 'System is not Linux')
+    def test_astralinuxce_2_os_grains(self):
```

Comment:
The value of `osrelease` is `2.12.22`, but I don't think we should use it for the testname.

Reply:
Seems reasonable enough for me. If we need to get more specific down the road, we can handle things then.